### DISCUSSION TITLE
Month Challenges? and Toonsquid social media Manager?

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#544] [Ideas]
I feel that while you are working on the next update that it would be cool if someone else can do the social media for you (like me) so to keep people updated and aware of Toonsquid. It does not have to be updates added to social media; it could just be events or challenges. Callipeg has a monthly challenge where you animate a topic on Callipeg. I feel it would be cool to do that for Toonsquid as it would eliminate the need for people to want constant updates since people have something to do and it would grow awareness of Toonsquid since people would see what it could do and people would love to participate since it brings awareness to them. I feel that I could do that job. If you are interested please let me know. I don't need to be paid since this is what I want to do, but if you want you can.

**Comment by Cr8iveMike** [comment]
I support this notion too....I'd also said something about this before...I think I gave it as an advice to periodically setup a challenge competition between users and interested newbies but nothing was done on it. I'm happy someone is catching up!

**Reply by Cat-Butler** [reply]
There is no reason challenges or social media postings need to be an "officially approved" channel.

For example, Ittai Manero posts monthly Procreate challenges on both the Procreate forum and on [his Xwitter account](https://twitter.com/ittaiman?lang=en). They are not Procreate-team official or approved, but there are some great entries.

If you feel strongly about it, go for it!

**Reply by Lioncub901** [reply]
Took your advice and made an Twitter and Instagram page for Toonsquid.

Twitter: https://twitter.com/ToonsquidCommu
Instagram: https://www.instagram.com/dinoball901/?next=%2F

Hopefully it can get enough followers so that we can make challenges.
